CMakeLists.txt

```diff
@@ -77,7 +77,7 @@ endif()
 
 set(DEFAULT_TARGET_PROPS
     # Require C++17.
-    CXX_STANDARD 17
+    CXX_STANDARD 20
```

Comment:
I don't think I like this, since this could allow using C++20 features accidentally when C++20 isn't required yet.